### PR TITLE
Added functionality to show subtree up to a certain level. Fixes #851

### LINKED ISSUE #851: FeatureModelEditor: collapse tree on tree-level basis
### Issue description

I would like to ask if it is possible to implement a functionality, that collapses the tree to a certain level, starting from a selected feature or root feature. A keyboard shortcut would be nice too.

There are other tree-structure editor that have similiar functionalities. For example Mindmanager from Mindjet/Microsoft. Even though this isn't very urgent, but at least this would be useful for me and my team to inspect the feature model.

### PR DESCRIPTION
**Pull request by sjvie**
Fixes #851

### CODE REVIEW

File: plugins/de.ovgu.featureide.fm.core/src/de/ovgu/featureide/fm/core/base/IFeatureStructure.java [hunk 1]
```diff
@@ -93,6 +93,13 @@
 
 	boolean isRoot();
 
+	/**
+	 * Calculates the depth of the subtree starting from this feature. If this feature has no children, the depth is 0.
+	 * 
+	 * @return The depth of the subtree starting from this feature
+	 */
+	int getSubtreeDepth();
```
Comment (Subaro) [inline plugins/de.ovgu.featureide.fm.core/src/de/ovgu/featureide/fm/core/base/IFeatureStructure.java:101]: @h3ssto Is the change to the interface required? It is only used one and seems to be more like an implementation for a FeatureStructureUtils or directly where it is used. When we change the interface then all implementations need to be adjusted.

**Comment by Subaro** [comment]
I just restarted the build on Travis as they failed again because of the GUI tests

File: plugins/de.ovgu.featureide.fm.core/src/de/ovgu/featureide/fm/core/base/IFeatureStructure.java (hunk 1 above)
Comment (h3ssto) [reply plugins/de.ovgu.featureide.fm.core/src/de/ovgu/featureide/fm/core/base/IFeatureStructure.java:101]: Yes, I also think this is unnecessary.

**Comment by h3ssto** [comment]
GUI Tests are running through on two of my machines and a Windows VM, merging this for now.

**Pull request merged**